### PR TITLE
Feature/dhp 3 models mogrations

### PR DESCRIPTION
**Pull request by peratik**

### CODE REVIEW

File: advanced_survey/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            ],
+        ),
+        migrations.CreateModel(
+            name='Question',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('page', models.IntegerField(db_index=True, default=0)),
+                ('list_order', models.IntegerField(db_index=True, default=0)),
+                ('question', models.TextField()),
+                ('comment', models.TextField(blank=True, null=True)),
```
Comment (David-Owen-Walker) [inline advanced_survey/migrations/0001_initial.py]: I think `description` is a better name

File: advanced_survey/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                ('field_type', models.CharField(choices=[('LineEdit', 'Line Edit'), ('TextArea', 'Text Area'), ('Select', 'Select'), ('Checkbox', 'Checkbix'), ('Radiobox', 'Radiobox'), ('Email', 'Email'), ('URL', 'URL'), ('Phone', 'Phone'), ('Boolean', 'Boolean'), ('Numeric', 'Numeric'), ('File', 'File'), ('Image', 'Image'), ('Date', 'Date'), ('Time', 'Time'), ('Rating', 'Rating')], max_length=255)),
+                ('options_url', models.URLField(blank=True, null=True)),
+                ('options', models.TextField(blank=True, null=True)),
+                ('is_required', models.CharField(blank=True, max_length=255, null=True)),
+                ('is_visible', models.CharField(blank=True, max_length=255, null=True)),
+                ('validation_rules', models.CharField(blank=True, max_length=255, null=True)),
```
Comment (David-Owen-Walker) [inline advanced_survey/migrations/0001_initial.py]: Could this be a regex? If so, a longer field length might be better

**Review by David-Owen-Walker — CHANGES_REQUESTED** [review]

File: advanced_survey/migrations/0001_initial.py (hunk 1 above)
Comment (peratik) [reply advanced_survey/migrations/0001_initial.py]: edited

File: advanced_survey/migrations/0001_initial.py (hunk 2 above)
Comment (peratik) [reply advanced_survey/migrations/0001_initial.py]: edited

**Review by David-Owen-Walker — APPROVED** [review]

**Pull request merged**